### PR TITLE
GSoC 2024 news

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: content/news/xxxx-xx-xx-mixxx-accepted-for-gsoc-2024.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+title: Mixxx accepted for GSoC 2024!
+authors: Daniel Schürmann
+tags: gsoc, gsoc-2024
+status: draft
+comments: yes
+
+Mixxx has been [accepted as a mentoring organization](https://summerofcode.withgoogle.com/programs/2024/organizations/mixxx) for [Google Summer of Code 2024](https://summerofcode.withgoogle.com/), a global program focused on bringing new contributors into open source software development. This is a great opportunity for us to attract new contributors, as they will receive a stipend sponsored by Google.
+
+How Does GSoC Work?
```
Comment (ronso0) [inline content/news/xxxx-xx-xx-mixxx-accepted-for-gsoc-2024.md]: Let's use headers.

`### How Does GSoC Work?`
or
`#### How Does GSoC Work?`

**Review by ronso0 — CHANGES_REQUESTED** [review]

**Comment by daschuer** [comment]
Done

**Comment by ronso0** [comment]
We have the same situation like with the waveforms post: the first header appears in the teaser.
image

We could move (and modify) the "You could also help us, even if.." paragraph up so the header is not shown.
Or we could ignore it :man_shrugging: 
Or someone knows from the top of his head where to adjust the teaser line treshold (or whatever it's called) to chop headers (and following lines)

Given we want to get this out soonish I'll wait for tomorrow afternoon (~14hours from now), then merge if there are no objections.
Note I didn't do an intense proof-read. LGTM

**Comment by daschuer** [comment]
I have discovered the "summary" field that solves our headaches with the headers.

**Comment by daschuer** [comment]
grafik

**Comment by ronso0** [comment]
> I have discovered the "summary" field that solves our headaches with the headers.

Great! Where can we add that info for authors?

**Comment by ronso0** [comment]
Okay, I'll merge this now. Thank you for taking care!

**Pull request merged**